rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -386,9 +386,9 @@ public Response getNestedLRAStatus(@PathParam("NestedLraId")String nestedLraId)
         LRAStatus status = lra.getLRAStatus();
 
         if (status == null || lra.getLRAStatus() == null) {
-            LRALogger.i18nLogger.error_cannotGetStatusOfNestedLraURI(nestedLraId, lra.getId());
-            String errMsg = String.format("LRA (parent: %s, nested: %s) is in the wrong state for operation " +
-                    "'getNestedLRAStatus': The LRA is still active.", nestedLraId, lra.getId());
+            String logMsg = LRALogger.i18nLogger.error_cannotGetStatusOfNestedLraURI(nestedLraId, lra.getId());
+            String errMsg = String.format("%s, LRA (parent: %s, nested: %s) %s", logMsg, nestedLraId, lra, "is in the wrong state for operation 'getNestedLRAStatus'.");
+            LRALogger.logger.error(errMsg);
```

Comment:
errMsg is not the same as the i18n log message (error_cannotGetStatusOfNestedLraURI) which it should be.